### ISSUE TITLE
RootFS is currently Read Only (R/O) mounted. Aborting...

### ISSUE THREAD

**Issue by Adsouza98** [MichaIng/DietPi#3511]
ADMIN EDIT:

A possible workaround could be:

1. login as `root`
2. `mount -o remount,rw /`
3. `nano /etc/fstab`
4. add the line for rootFS > `PARTUUID=<YOUR-ID>-02 / ext4 noatime,lazytime,rw 0 1`
5. `PARTUUID` needs to be adjusted, should be same like `/boot` if you use SD card. Be careful and use the right partition `-02`
6. save and exit
7. `reboot`

To get/verify the ID just use `blkid`. You can identify the rootFS by the lable `LABEL="rootfs"`

```
root@DietPi3:~# blkid
/dev/mmcblk0p1: LABEL_FATBOOT="boot" LABEL="boot" UUID="5203-DB74" TYPE="vfat" PARTUUID="6c586e13-01"
/dev/mmcblk0p2: LABEL="rootfs" UUID="2ab3f8e1-7dc6-43f5-b0db-dd5759d51d4e" TYPE="ext4" PARTUUID="6c586e13-02"
/dev/mmcblk0: PTUUID="6c586e13" PTTYPE="dos"
```

It should looks like this afterwards (just with your PARTUUID)
```
#----------------------------------------------------------------
# PHYSICAL DRIVES
#----------------------------------------------------------------
PARTUUID=6c586e13-02 / ext4 noatime,lazytime,rw 0 1
PARTUUID=6c586e13-01 /boot vfat noatime,lazytime,rw 0 2
```

---

#### Required Information
- DietPi version | v6.29.2
- Distro version | buster
- Kernel version | Linux DietPi 4.19.97-v7l+ #1294 SMP Thu Jan 30 13:21:14 GMT 2020 armv7l GNU/Linux
- SBC model | RPi 4 Model B (armv7l)
- Power supply used | 5V 1A RAVpower
- SDcard used | 32 GB SanDisk ultra

#### Additional Information (if applicable)
- The software title installed freshly
- Can this issue be replicated on a fresh installation of DietPi? **YES**.

#### Steps to reproduce

1. Installed Fresh version of DietPi from website
2. Flashed SD Card with image
3. Instlled Sonarr, Nzbget, Deluge
4. Set a static IP address
6. Installed Open VPN client with `sudo apt-get install -y openvpn`
7. mounted my NAS via NFS

#### Expected behaviour
- Running as usual 
- Softwares working

#### Actual behaviour
-  After rebooting RootFS becomes Read only
- All Software stops working
- Unable to use dietpi-_______ commands except for dietpi-explorer and dietpi-drive_manager
- When I try and run dietpi-Software for example I get:

> RootFS is currently Read Only (R/O) mounted. Aborting
>DietPi requires RootFS to be Read/Write (R/W) mounted. Please run "dietpi-drive_manager" to re-enable.

![image](https://user-images.githubusercontent.com/25352861/81137306-5ffd7300-8f2c-11ea-90e0-ce4ca2db2b69.png)

When I run dietpi-drive_manager it notices all mounted drives but at the end it says:
> cp: cannot create regular file '/etc/fstab': Read-only file system

![image](https://user-images.githubusercontent.com/25352861/81137349-7dcad800-8f2c-11ea-975a-778ebd323f41.png)

in the dietpi-drive_manager it says
> Please select a drive to see available options.
>      - User data location: RootFS (/mnt/dietpi_userdata)

![image](https://user-images.githubusercontent.com/25352861/81137383-9fc45a80-8f2c-11ea-9e0e-0e927be690f5.png)

![image](https://user-images.githubusercontent.com/25352861/81137401-b10d6700-8f2c-11ea-8b7a-8e4d92b896af.png)

When I choose the `/boot` option and select `Transfer RootFS` it says:

> It says [WARNING] Partition must be unmounted, before format and RootFS transfer can begin.
Please unmount the partition, then try again.

![image](https://user-images.githubusercontent.com/25352861/81137419-c2ef0a00-8f2c-11ea-8413-ba03971ad937.png)

I tried doing a fresh install 3 times and still have the same problem. This only started occurring after updating was working fine before.

Any help to fix this will be appreciated.

**Comment by zegxro** [comment]
Same here.. at 1st it happen when ever i install plex media server.. but after a while.. it seem to happen just after i freshly installed dietpi..

I also notice it will change RootFS to read-only after the swap failed.. when you try to mount using dietpi-drive manager.. sometimes it will mount.. but most of the time it wont mount no matter how many time you try using dietpi-drive manager..

The only way to mount it is by using [ mount -o remount,rw / ] but.. after a reboot.. RootFS will change back to read-only stat..

Been struggling the whole day since yesterday but failed to find permanent solution.. change 3 new SD Card with different size but same result.. tested on 2 raspberry pi version - raspberry pi v2 n v3 both have the same error.

Update : Forget to mention that the Network Time Synchronization also failed..

**Comment by sealteam666** [comment]
same happened with I changed the hostname from dietpi to pi then i rebooted then it  changed to rootfs readonly with same error as above all the software stopped working also i updated it yesterday from 6.28 to 6.29.2..

please help

**Comment by dasrocky** [comment]
I faced exactly same issue too, This happens only when update/upgrade from v6.28 to v6.29.2.
I solved this issue with clean installation. (Honestly, i didn't face any issue with rootfs thingy after clean installation)(Might help, **read issue #3510 comment section**, I mentioned a workaround, although it is for wifi hardware, might work for roots) worked for me

(Warning) The problem with a clean installation, that if u use wifi instead of ethernet. It will mention no WIFI hardware found. I have already posted an issue for that, hoping for some fixes. By meantime I found a workaround for the wifi issue, u can check in Issues section

**Comment by Joulinar** [comment]
Hi Guys,

looks like it's not happen to everybody and it's doesn't seems to have a pattern. Just for information, pls can you have a look to `/etc/fstab` how the rootFS mount entry looks like?

EDIT: Are you all using RPi devices or some other SBC's as well??

**Comment by dasrocky** [comment]
> Hi Guys,
> 
> looks like it's happen to everybody and it's doesn't seems to have a pattern. Just for information, pls can you have a look to `/etc/fstab` how the rootFS mount entry looks like?

Could u pls mention what, exactly u need from fstab file?

using rpi3 b+. But somehow it is working now for me.

**Comment by Joulinar** [comment]
Hi,

in theory it should contain these 2 lines at the end of the file for rootFS as well as bootFS

```
#----------------------------------------------------------------
# PHYSICAL DRIVES
#----------------------------------------------------------------
PARTUUID=6c586e13-02 / ext4 noatime,lazytime,rw 0 1
PARTUUID=6c586e13-01 /boot vfat noatime,lazytime,rw 0 2

```

**Comment by dasrocky** [comment]
> Hi,
> 
> in theory it should contain these 2 lines at the end of the file for rootFS as well as bootFS
> 
> ```
> #----------------------------------------------------------------
> # PHYSICAL DRIVES
> #----------------------------------------------------------------
> PARTUUID=6c586e13-02 / ext4 noatime,lazytime,rw 0 1
> PARTUUID=6c586e13-01 /boot vfat noatime,lazytime,rw 0 2
> ```


PARTUUID=6c586e13-02 / ext4 noatime,lazytime,rw 0 1
PARTUUID=6c586e13-01 /boot vfat noatime,lazytime,rw 0 2
/var/swap none swap sw 0 0
tmpfs /var/log tmpfs size=50M,noatime,lazytime,nodev,nosuid,mode=1777
/dev/sda2 /mnt/ssd ntfs-3g defaults 0 0
/dev/sdb2 /mnt/ssd ntfs-3g defaults 0 0
/dev/sdc2 /mnt/ssd ntfs-3g defaults 0 0

Last 3 lines are added by me, for mounting drives after boot

Thank you,

**Comment by Joulinar** [comment]
that's fine but if I understood right, your installation is working now. So we need a failed setup to compare.

**Comment by dasrocky** [comment]
> that's fine but if I understood right, your installation is working now. So we need a failed setup to compare.

Yes, presently. I am w8ing too for comparison

**Comment by Joulinar** [comment]
OK guys, I was able to produce the error by using `dietpi-drive_manager`. On my test case, drive manager was removing the line responsible for the rootFS

`PARTUUID=6c586e13-02 / ext4 noatime,lazytime,rw 0 1`

It was simply not there anymore. After reboot the rootFS was ro and all stop working. I got it back as follow

1. login as `root`
2. `mount -o remount,rw /`
3. `nano /etc/fstab`
4. add the line for rootFS > `PARTUUID=<YOUR-ID>-02 / ext4 noatime,lazytime,rw 0 1`
5. `PARTUUID` needs to be adjusted, should be same like `/boot` if you use SD card. Be careful and use the right partition `-02`
6. save and exit
7. `reboot`

**Comment by Daniel-Myers** [comment]
My setup currently isn't working. My etc/fstab contains
`PARTUUID=6c586e13-01 /boot vfat noatime,lazytime,rw 0 2
#UUID=2ab3f8e1-7dc6-43f5-b0db-dd5759d51d4e /mnt/2ab3f8e1-7dc6-43f5-b0db0dd5759d51d4e ext4 [noatime,lazytime,rw,nofail,noauto,x-systemd.automount`
![2020-05-06 11_42_34-Window](https://user-images.githubusercontent.com/57239143/81168370-c87d3c00-8f8e-11ea-8fbc-348533b87e42.png)
Should I still try to do the fix you mentioned in https://github.com/MichaIng/DietPi/issues/3511#issuecomment-624525544

**Comment by zegxro** [comment]
> OK guys, I was able to produce the error by using `dietpi-drive_manager`. On my test case, drive manager was removing the line responsible for the rootFS
> 
> `PARTUUID=6c586e13-02 / ext4 noatime,lazytime,rw 0 1`
> 
> It was simply not there anymore. After reboot the rootFS was ro and all stop working. I got it back as follow
> 
> 1. login as `root`
> 2. `mount -o remount,rw /`
> 3. `nano /etc/fstab`
> 4. add the line for rootFS > `PARTUUID=<YOUR-ID>-02 / ext4 noatime,lazytime,rw 0 1`
> 5. `PARTUUID` needs to be adjusted, should be same like `/boot` if you use SD card. Be careful and use the right partition `-02`
> 6. save and exit
> 7. `reboot`

Testing it.. so far so good.. but i need to test it with plex media server + NAS drive mounted.. will post back ASAP..

**Comment by Joulinar** [comment]
@Daniel-Myers 
yes pls try the workaround I posted above https://github.com/MichaIng/DietPi/issues/3511#issuecomment-624525544

@zegxro 
be careful while suing drive manager. It could lead to same again

**Comment by zegxro** [comment]
> @Daniel-Myers
> yes pls try the workaround I posted above [#3511 (comment)](https://github.com/MichaIng/DietPi/issues/3511#issuecomment-624525544)
> 
> @zegxro
> be careful while suing drive manager. It could lead to same again

got it.. but if it happen again.. i just need to add [ `PARTUUID=<YOUR-ID>-02 / ext4 noatime,lazytime,rw 0 1` ] again ?..

**Comment by Joulinar** [comment]
yes, that would be the workaround for the time being until we have a permanent fix

**Comment by Daniel-Myers** [comment]
> 
> 
> @Daniel-Myers
> yes pls try the workaround I posted above [#3511 (comment)](https://github.com/MichaIng/DietPi/issues/3511#issuecomment-624525544)

Thanks, that's fixed it for now. Do have any idea why that line get's removed?

**Comment by Joulinar** [comment]
probably a bug inside the new release. Would need to be verified by the developer.

**Comment by richard-downes** [comment]
Just to help or not. I'm just rebooting after your fix.

I have had this happen now on 3 separate RPi3s. I have some spares, and thought it was a hardware problem with the first one. Then tried multiple cards as well. 8gb/16gb/32gb from different manufacturers (scandisk/toshiba). Forgot I tried it on a RPi4 as well. yesterday.

So far your fix has worked. Just rebooted. I'm going to open fstab and copy/paste the same like with a comment so its easier to put in next time.

**Comment by UlrikWKoren** [comment]
Hi, i get this error after installing nextcloud with external hdd mounted on rpi4.
im wondering where do i find my rootFS id to put in the fstab file?

**Comment by ludovickk** [comment]
The problem occurs right after using DietPi-Drive_Manager.

**Comment by Joulinar** [comment]
@ludovickk 
Yes known. A workaround is provided above

**Comment by LeVraiRoiDHyrule** [comment]
Hi, I've got the same issue but I can't figure out how to apply the solution. I'm new and I don't know how to manage the fstab file. Here is mine : 

https://pastebin.com/Nhx8Zr4T

I can't find what I should write instead of YOUR-ID .
I don't know how to find this ID. Can someone help me ? Sorry if I'm a noob. Any answer is welcome and have a good day

**Comment by UlrikWKoren** [comment]
> Hi, I've got the same issue but I can't figure out how to apply the solution. I'm new and I don't know how to manage the fstab file. Here is mine : 
> 
> 
> 
> https://pastebin.com/Nhx8Zr4T
> 
> 
> 
> I can't find what I should write instead of YOUR-ID .
> 
> I don't know how to find this ID. Can someone help me ? Sorry if I'm a noob. Any answer is welcome and have a good day

Im wondering the same, i Can’t find the ID.

**Comment by Daniel-Myers** [comment]
> 
> 
> Hi, I've got the same issue but I can't figure out how to apply the solution. I'm new and I don't know how to manage the fstab file. Here is mine :
> 
> https://pastebin.com/Nhx8Zr4T
> 
> I can't find what I should write instead of YOUR-ID .
> I don't know how to find this ID. Can someone help me ? Sorry if I'm a noob. Any answer is welcome and have a good day

The ID is the same as the previous PARTUUID, so it is 6c586e13, that's what I did anyway and it worked. This may be different for different people.

**Comment by UlrikWKoren** [comment]
> 
> 
> 
> 
> 
> 
> > 
> 
> > 
> 
> > Hi, I've got the same issue but I can't figure out how to apply the solution. I'm new and I don't know how to manage the fstab file. Here is mine :
> 
> > 
> 
> > https://pastebin.com/Nhx8Zr4T
> 
> > 
> 
> > I can't find what I should write instead of YOUR-ID .
> 
> > I don't know how to find this ID. Can someone help me ? Sorry if I'm a noob. Any answer is welcome and have a good day
> 
> 
> 
> The ID is the same as the previous PARTUUID, so it is 6c586e13, that's what I did anyway and it worked. This may be different for different people.

Thanks, i will try when i get home😊

**Comment by LeVraiRoiDHyrule** [comment]
> 
> 
> > Hi, I've got the same issue but I can't figure out how to apply the solution. I'm new and I don't know how to manage the fstab file. Here is mine :
> > https://pastebin.com/Nhx8Zr4T
> > I can't find what I should write instead of YOUR-ID .
> > I don't know how to find this ID. Can someone help me ? Sorry if I'm a noob. Any answer is welcome and have a good day
> 
> The ID is the same as the previous PARTUUID, so it is 6c586e13, that's what I did anyway and it worked. This may be different for different people.

I would like to try that but I've got another issue : I can't edit my fstab file, it's protected and read-only

![image](https://user-images.githubusercontent.com/64923542/81216566-672f8a00-8fdb-11ea-89af-8c31d84a97e4.png)
How can I unlock it ?

**Comment by Joulinar** [comment]
Guys it depends on your setup. If you use SD card only, than the ID is same as used for `/boot`. In my case it's like this. Both ID are same except the `-02` at the end

```
PARTUUID=6c586e13-02 / ext4 noatime,lazytime,rw 0 1
PARTUUID=6c586e13-01 /boot vfat noatime,lazytime,rw 0 2
```

However if you transfered your rootFS to an external device like USB HDD, the ID will be different than `/boot`

You can use `lsblk` to display our USB devices and to get the ID just use `blkid`

That's how it will looks like. You can identify the rootFS by the lable `LABEL="rootfs"`

```
root@DietPi3:~# blkid
/dev/mmcblk0p1: LABEL_FATBOOT="boot" LABEL="boot" UUID="5203-DB74" TYPE="vfat" PARTUUID="6c586e13-01"
/dev/mmcblk0p2: LABEL="rootfs" UUID="2ab3f8e1-7dc6-43f5-b0db-dd5759d51d4e" TYPE="ext4" PARTUUID="6c586e13-02"
/dev/mmcblk0: PTUUID="6c586e13" PTTYPE="dos"
```

**Comment by UlrikWKoren** [comment]
> > 
> 
> > 
> 
> > > Hi, I've got the same issue but I can't figure out how to apply the solution. I'm new and I don't know how to manage the fstab file. Here is mine :
> 
> > > https://pastebin.com/Nhx8Zr4T
> 
> > > I can't find what I should write instead of YOUR-ID .
> 
> > > I don't know how to find this ID. Can someone help me ? Sorry if I'm a noob. Any answer is welcome and have a good day
> 
> > 
> 
> > The ID is the same as the previous PARTUUID, so it is 6c586e13, that's what I did anyway and it worked. This may be different for different people.
> 
> 
> 
> I would like to try that but I've got another issue : I can't edit my fstab file, it's protected and read-only
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/64923542/81216566-672f8a00-8fdb-11ea-89af-8c31d84a97e4.png)
> 
> How can I unlock it ?

Write this first in command line, then try to Edit FSTab $ mount -o remount,rw /

**Comment by Joulinar** [comment]
@LeVraiRoiDHyrule 
did you have done `mount -o remount,rw /` before as described on the workaround?

**Comment by LeVraiRoiDHyrule** [comment]
No I didn't, sorry I'm just dumb.......   I did it now. Sorry for disturbing.

**Comment by Joulinar** [comment]
nothing to excuse, we are there to help 😉

**Comment by LeVraiRoiDHyrule** [comment]
@Joulinar It's fixed, thanks !!!! As I didn't modified anything about the rootfs partition, it was the same ID as yours. Thank you so much for your patience and your help. Have a good day !!!

**Comment by Chublo** [comment]
Ok, so it appears I made an error when putting the PARTUUID into FSTAB. Now I can't run mount -o remount,rw / so that I can edit the file again. Any ideas?

root@DietPi:~# mount -o remount,rw /
mount: /: can't find PARTUUID=cb7b86f7-01-02.

**Comment by Joulinar** [comment]
@Chublo 
do you have stored your rootFS on an external device or still using SD card? Pls can you provide output of `lsblk` as well as `blkid`

**Comment by Chublo** [comment]
Still on SD

```
root@DietPi:~# lsblk
NAME        MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
mmcblk0     179:0    0  7.4G  0 disk 
├─mmcblk0p1 179:1    0  256M  0 part /boot
└─mmcblk0p2 179:2    0  7.1G  0 part /
```


```
root@DietPi:~# blkid
/dev/mmcblk0p1: LABEL_FATBOOT="boot" LABEL="boot" UUID="F661-303B" TYPE="vfat" PARTUUID="cb7b86f7-01"
/dev/mmcblk0p2: LABEL="rootfs" UUID="8d008fde-f12a-47f7-8519-197ea707d3d4" TYPE="ext4" PARTUUID="cb7b86f7-02"
/dev/mmcblk0: PTUUID="cb7b86f7" PTTYPE="dos"
root@DietPi:~#
 ```

**Comment by Joulinar** [comment]
ok issue is that you have `-01-02` at the end of your ID but it should be `-02` only

just try following
```
mkdir /mnt/temp
mount /dev/mmcblk0p2 /mnt/temp
nano /mnt/temp/etc/fstab
```

**Comment by Chublo** [comment]
Read-only file system :(

`root@DietPi:~# mkdir /mnt/temp
mkdir: cannot create directory ‘/mnt/temp’: Read-only file system
root@DietPi:~# 
`

**Comment by Joulinar** [comment]
ok can you create something on `/boot` like

`mkdir /boot/temp`

**Comment by Chublo** [comment]
That works, but can't mount to it - already mounted.
```
root@DietPi:~# mkdir /boot/temp
root@DietPi:~# mount /dev/mmcblk0p2 /boot/temp
mount: /boot/temp: /dev/mmcblk0p2 already mounted on /.
root@DietPi:~# 
```

**Comment by reeslo** [comment]
Today I've updated 2 Pi `RPi B+` and `RPi 3 B`

I have the issue on RPi 3 B only, just after a power outage (due to improper fan pin placement :/).
Just after reboot I see RootFS was ro :'(

I thought of a problem following the update, I rebooted the RPi B, and I did not notice the issue.


For me this line was not present into /etc/fstab, and above solution, resolve it :)
```
PARTUUID=6c586e13-02 / ext4 noatime,lazytime,rw 0 1
```
Thanks !

**Comment by Joulinar** [comment]
@Chublo 
do you have another Linux box like a different DietPi system? Or a system that can read ext4 file systems?

**Comment by Chublo** [comment]
> That works, but can't mount to it - already mounted.
> 
> ```
> root@DietPi:~# mkdir /boot/temp
> root@DietPi:~# mount /dev/mmcblk0p2 /boot/temp
> mount: /boot/temp: /dev/mmcblk0p2 already mounted on /.
> root@DietPi:~# 
> ```
The below solved the issue for me and enabled me to edit the FSTAB again to correct the error.  Thanks @Joulinar for your help.
`root@DietPi:/boot/temp# sudo mount -o remount,rw /dev/mmcblk0p2 /`

**Comment by Adsouza98** [comment]
Fix Worked! Thank you for the help @Joulinar. Mad respects to moderators and Github forms!

Please update this thread when there is a proper solution in a new update.

Thanks again.

**Comment by loris-av** [comment]
New reboot today, issue repeated :'(

**Comment by Joulinar** [comment]
It could happen again if you use DietPi Drive Manager as long as there is no fix on the DietPi script's

**Comment by UlrikWKoren** [comment]
Won’t the problem occour if you mount your drives manually in fstab? Instead of using dietpi-drive_manager.

**Comment by Joulinar** [comment]
nope, this should not happen. During my investigation I just found that Drive Manager is deleting the line. If you edit `/etc/fstab` directly it should be ok. I mean this is the way how we implement the workaround as well 😉

**Comment by mamton** [comment]
Joulinar it is a continuous problem. Every time you reboot the same problem comes up. Anytime the RPi is rebooted you have to mount/remount and adjust the fstab file. I wouldn't really call this a workaround as the problem never goes away really, every reboot the problem starts again.

**Comment by Chublo** [comment]
I’ve rebooted and the issue is still resolved.

**Comment by dunxd** [comment]
I'm getting the same issue with a Raspberry Pi 2 after running dietpi-update.  Was using this for Pihole, which doesn't work until the root partition is set to writeable. The workaround is working for me.

**Comment by ElTav** [comment]
Also running into this same issue as well. @MichaIng any idea what the issue might be?

**Comment by Joulinar** [comment]
@ElTav 
Did you used Drive Manager?

**Comment by reeslo** [comment]
I confirm, issue appear every time after use `dietpi-drive_manager`.
And sometime there is problems to mount new disks.

For me issue source is `dietpi-drive_manager`

**Comment by campaquino** [comment]
Hey guys, I took a look into the dietpi-drive_manager script. It seems it is using the global variable G_ROOTFS_DEV due to a recent change. But i cannot find any definition for it, neither in the script itself, nor in the called dietpi-globals script.

**Comment by reeslo** [comment]
@Chublo try to mount SD card to another computer and open `/etc/fstab` file as root (from `boot` drive I think).

**Comment by MichaIng** [comment]
Found and fixed the bug... nasty thing that on RPi `df` prints /dev/root for the rootfs mount source, which does not exist and hence no info on that mount was estimated: https://github.com/MichaIng/DietPi/commit/1570e47acac88414ecc8586845f24e70fb38461b

Due to circumstances I was not able to test the latest changes on native RPi. Was the very last change before DietPi v6.29 release, hence slipped through beta. Midterm we really need some clear testing schedule or even automated CI/testing. The problem just is that the very important stuff is based on the physical machine (like exactly this issue on RPi only), which cannot be covered by CI providers.

However, I'll go through the last issue with missing DietPi script files and push a hotfix release as fast as possible.

**Comment by Joulinar** [comment]
@MichaIng 
Was already thinking the same to create a test plan/scenario or to create some kind of checklist for most common features. But yeah something to develop.

**Comment by AstroTaco84** [comment]
Thank you all. New to rpi4 and ran into this few days ago. Work around fix provided worked.

**Comment by Callumro** [comment]
Hey all, I wanted to note the workaround seems like a temporary fix. My FS keeps reverting to read-only

**Comment by WolfShadowheart** [comment]
> 
> 
> OK guys, I was able to produce the error by using `dietpi-drive_manager`. On my test case, drive manager was removing the line responsible for the rootFS
> 
> `PARTUUID=6c586e13-02 / ext4 noatime,lazytime,rw 0 1`
> 
> It was simply not there anymore. After reboot the rootFS was ro and all stop working. I got it back as follow
> 
>     1. login as `root`
> 
>     2. `mount -o remount,rw /`
> 
>     3. `nano /etc/fstab`
> 
>     4. add the line for rootFS > `PARTUUID=<YOUR-ID>-02 / ext4 noatime,lazytime,rw 0 1`
> 
>     5. `PARTUUID` needs to be adjusted, should be same like `/boot` if you use SD card. Be careful and use the right partition `-02`
> 
>     6. save and exit
> 
>     7. `reboot`

Hello, new to the OS and so my familiarity with certain things is lacking. I've been having a heck of a time with this update because I was already having issues permanently mounting USB drives. 

I have just done another fresh flash of DietPi because I put the OS into emergency mode trying to properly configure this workaround. I have a 128GB SD card so my line shows vfat instead of ext4, I'm not sure if that has something to do with it. 

So I am very unsure as to where I'm supposed to be adding all of these lines. Upon a fresh install of dietpi, mounting drives via drive manager so that I can permanently mount my 2 hard drives, I can navigate to 

nano /etc/fstab  

and can see that I have one line

PARTUUID=6C586E13-02 /boot vfat noatime,lazytime,rw 0 1

and it seems that adding another instance above or below the line doesn't do much (like so): 

PARTUUID=6C586E13-01 /boot vfat noatime,lazytime,rw 0 1

If possible, I'd very much appreciate some guidance for how I can permanently mount my 2 USB hard drives, I'm going crazy and becoming a tad frustrated :( 

This is what I've got in my fstab 

__________________________
PHYSICAL DRIVES

PARTUUID=6C586E13-01 /boot vfat noatime,lazytime,rw 0 1
PARTUUID=6C586E13-02 /boot vfat noatime,lazytime,rw 0 1
UUID=F4A0F077A0F041A2 /mnt/F4A0F077A0F041A2 ntfs noatime,lazytime,rw,permissions,nofail,noauto,x-systemd.auto$

UUID=41428830-e1b0-8a2d-ca7bacca /mnt/41428830-e1b0-8a2d-ca7bacca ext4 noatime,lazytime,rw,permissions,nofail,noauto,x-systemd.auto$

#UUID=2ab3f8e1-7dc6-43f5-b0db-dd5759d51d4e /mnt/2ab3f8e1-7dc6-43f5-b0db-dd5759d51d4e ext4 noatime,lazytime,rw,permissions,nofail,noauto,x-systemd.auto$

UUID=F4A0F077A0F041A2 / auto defaults,noatime,rw 0 1
UUID=41428830-e1b0-44af-8a2d-ca7bacca9992 / auto defaults,noatime,rw 0 1

________________________

Each reboot I am told that /etc/fstab is unwritable, so I imagine I'm doing something wrong. 

I'm at a loss of what to do, I see that #3511 is the answer however I am not understanding how to use it to resolve my problem and keep these hard drives permanently mounted. 

Thanks a bunch for any help offered.

**Comment by mistakenideas** [comment]
Thanks for the workaround. 

I’d tested the Beta (although only briefly) on a physical RPi (an original Model B Rev 1) and not hit this problem, it is still on the beta branch and the contents of /etc/fstab is fine. The bug only arose on my other newer RPi after running a few days and rebooting after some other updates. Presumably this bug only occurs under circumstances which trigger the dietpi-drivemanager script to run? I hadn’t manually done so. 

Update: I had manually run dietpi-drive_manager between updating and rebooting. My bash history has a better memory than me :)

I’ve also confirmed I can reproduce the problem on my other Pi with the following steps (simply running dietpi-drive_manager is enough, which I guess you already know)
- Update to DietPi 6.29.2
- Check contents of /etc/fstab
- Run dietpi-drive_manager and exit immediately
- Check contents of /etc/fstab

On the next reboot the root partition will me mounted as read-only resulting in various problems. 

Physical Drives section of /etc/fastab before

```
#----------------------------------------------------------------
# PHYSICAL DRIVES
#----------------------------------------------------------------
PARTUUID=6c586e13-02 / ext4 noatime,lazytime,rw 0 1
PARTUUID=6c586e13-01 /boot vfat noatime,lazytime,rw 0 2
```

Physical Drives section of /etc/fstab after
```
#----------------------------------------------------------------
# PHYSICAL DRIVES
#----------------------------------------------------------------
PARTUUID=6c586e13-01 /boot vfat noatime,lazytime,rw 0 2
#UUID=2ab3f8e1-7dc6-43f5-b0db-dd5759d51d4e /mnt/2ab3f8e1-7dc6-43f5-b0db-dd5759d51d4e ext4 noatime,lazytime,rw,nofail,noauto,x-systemd.automount
```

**Comment by Joulinar** [comment]
Hi guys,

To all of you who use Drive Manager. DON'T DO THIS. Currently there is a bug affecting RPi devices only. Who did not like to wait for the permanent solution, a fix is already there within dev branch. Not sure if it was already merged into beta. Anyway, it will be released soon. Pls be patient .

**Comment by WolfShadowheart** [comment]
> 
> 
> Hi guys,
> 
> To all of you who use Drive Manager. DON'T DO THIS. Currently there is a bug affecting RPi devices only. Who did not like to wait for the permanent solution, a fix is already there within beta branch.

Don't do what? I'm sorry! 

As of now, is there any way to permanently mount USB drives? I have been using [this guide](https://gist.github.com/etes/aa76a6e9c80579872e5f) to attempt mounting hard drives without using drive manager. 

Currently I have gotten both of my hard drives mounted by creating /mnt/volume/1TB and /mnt/volume/4TB and then mounting the hard drives using `sudo mount /dev/sda1 /mnt/volume/1TB` and similarly `sudo mount /dev/sdb1 /mnt/volume/4TB` for each. 

Per my previous comment I couldn't get any of the UUID specifications to work where I've previously had success using the sda1 format. At this point I just want to make progress hehe :) 

However, now going into /etc/fstab they do not show up in the physical drives section, I would imagine because fstab is edited by driver mangager and thus hasn't been told to mount the drives. 

Is moving forward a waste of time until this update is released, or should I tinker around and try to get this functional? Unfortunately, my whole server is down until this can be fixed, as I haven't been able to get any of my media drives even mounted. 

Very sorry, new to linux. I've successfully written a Magic Mirror module for RPi, but cannot mount a USB hard drive to one. :) Any help is appreciated! Thank you.

**Comment by Joulinar** [comment]
Hi,

> Don't do what? I'm sorry!

Don't use `dietpi-drive_manager` for the time being

you need to edit `/etc/fstab` manually to have your HDD mounted automatically. Just using `mount` command will not add anything to `/etc/fstab`

pls can you provide output of  following

```
lsblk -Sfo +size
lsblk 
blkid
cat /etc/fstab
```

**Comment by WolfShadowheart** [comment]
Apologies, I will have to type it out as I don't have an easy way to screengrab sadly. Photos are an alternative, but I imagine text is best. What follows is from a fresh install of DietPi having never entered driver manager. The only thing I've done is put the display into portrait mode for my monitor, through dietpi-config display settings.

```
sda 0:0:0:0 disk Seatgate ST1000LM024_HN-M101MB 0836 usb 931.5G
sda 1:0:0:0 disk TOSHIBA External_USB_3.0 5438 usb 3.7T 
```

```
 sda 8:0 0 931.5G 0 disk
 sda1 8:1 0 931.5G 0 part
sdb 8:16 0 3.7T 0 disk
sdb1 8:17 0 3.7T 0 part
mmcblk0 179:0 0 119.1G 0 disk
mmcblk0p1 179:1 0 256M 0 part /boot
mmcblk0p2 179:2 0 118.9G 0 part /
```

```
/dev/mmcblk0p1: LABEL_FATBOOT="boot" LABEL="boot" UUID="5203-DB74" TYPE="vfat" PARTUUID="6c586e13-01"
/dev/mmcblk0p2: LABEL="rootfs" UUID="2ab3f8e1-7dc6-43f5-b0db-dd5759d51d4e" TYPE="ext4" PARTUUID="6c586e13-02"
/dev/sda1: LABEL="External HDD" UUID="F4A0F077A0F041A2" TYPE="ntfs" PARTUUID="5e432f0d-01"
/dev/sdb1: UUID="41428830-31b0-44af-8a2d-ca7bacca9992" TYPE="ext4" PARTLABEL="primary" PARTUUID="8731a3ff-486f-473d-b0db-b0b2b7269e9d"
```


For a copy of fstab, again noting I have not used drive manager yet on this install, and I've just now added the last 2 lines of /dev/sd## and rebooted with no luck. Having them there or not there has not affected anything on previous installs.  

Please use... 
Network (empty)
TMPFS
`tmpfs /tmp tmpfs size=1024m,noatimelazytime,nodev,nosuid,mode=1777`
Misc (empty)
Swapfile (empty)
```
PHYSICAL DRIVES
PARTUUID=6c586e13-02 / ext4 noatime,lazytime,rw 0 1
PARTUUID=6c586e13-01 /boot vfat noatime,lazytime,rw 0 2
/var/swap none swap sw 0 0
/var/log tmpfs size=50M,noatime,lazytime,nodev,nosuid,mode=1777
/dev/sda1 /mnt/volume/1TB ntfs-3g defaults,nofail,x-systemd.device-timeout=1,noatime 0 0
/dev/sdb1 /mnt/volume/4TB ext4 defaults,nofail,x-systemd.device-timeout=1,noatime 0 0
```

That's everything! Thanks a bunch for investigating. 

Also, if needed I am happy to add or remove whatever, or if you need more info on how drive manager affects fstab, I've done like 10 flashes so 10 more can't hurt :P 

Edit: on reboot is says it skips trying to mount, I'm using instructions from [this](https://gist.github.com/etes/aa76a6e9c80579872e5f) guide because I require guidance :) I am going to play with the device-timeout=# in case the hard drives aren't waking up before they can be mounted on boot. 

Edit: Alriiiiight nice so changing `/dev/sda1 /mnt/volume/1TB ntfs-3g defaults,nofail,x-systemd.device-timeout=1,noatime 0 0`
`/dev/sdb1 /mnt/volume/4TB ext4 defaults,nofail,x-systemd.device-timeout=1,noatime 0 0`

at the bottom of the fstab to 

`/dev/sda1 /mnt/volume/1TB ntfs-3g defaults,nofail,x-systemd.device-timeout=30,noatime 0 0`
`/dev/sdb1 /mnt/volume/4TB ext4 defaults,nofail,x-systemd.device-timeout=30,noatime 0 0`

I was met with a systemd error and a failed mount of the 1TB drive, BUT THE 4TB DRIVE AUTOMOUNTED! This is the first time this has happened for me since I received the drive 5 days ago, so the is a big breakthrough for me. 

There is likely a much better way to do this - I have to resort to online sources and using the arch linux guide has not proven as an effective tool for me unfortunately. 


Edit: First I apologize for any edits you have to make for formatting, I am not used to github and am used to Reddit. 

checking dmesg doesn't have anything related to that, it gives the same errors it has for as long as I've used this OS, failed to open VHCI service and shared memory service. . 


With my most recent update to /etc/fstab, upon rebooting I am told see systemctl status mnt-volume-1TB.mount and so I type that in and get this: a red square next to  

     mnt-volume-1TB.mount - /mnt/volume/1TB
     Loaded: loaded (/etc/fstab; generated)
     Active: failed (Result: exit-code) since today's date 2mins ago
     Where: /mnt/volume/1TB
     What: /dev/disk/by-uuid/F4A0F077A0F041A2
     Docs: man:fstab(5)
              man:systemd-fstab-generator(8)

The following lines all are preceeded by todays time and date

     DietPi systemd[1]: Mounting /mnt/volume/1TB. . . 
     DietPi mount[370]: mount: /mnt/volume/1TB: unknown filesystem type 'ntfs-3g'. 
     DietPi systemd[1]: mnt-volume-1TB.mount: Mount process exited, code=exited, status=32/n/a
     DietPi systemd[1]: mnt-volume-1TB.mount: Failed with result 'exit code'
     DietPi systemd[1]: mnt-volume-1TB.mount: Failed to mount /mnt/volume/1TB


I'm currently reinstalling ntfs-3g just in case it was missing, since that's what was indicated, and hoping the upon reboot that both drives are mounted automatically! I deeply apologize for all of the edits, although I figure they are better than posts and keep the information consolidated.  

Also, I just noticed in my typed up fstab here that I listed the hard drive as nfts-3g, as opposed to just ntfs. I have changed it, rebooted, and while nothing was specified upon the reboot, checking df -h upon reboot does show both hard drives available, so taking all of these steps one by one I eventually eliminated everything I had done incorrectly. 


From here - I want to ensure that I do not interact with drive manager, correct? 

I can make a copy of my fstab that I can use to restore it in case I do, or would it be better to write down my fstab contents? 

I've spent so long trying to get hard drives permanently mounted that I don't even remember what it was that I wanted to do with my Pi! 🐺 

I think I have resolved my problem with your work around, I very much appreciate your help. If you have any glaring issues you see in my code that you just couldn't bear to stand someone living with, feel free to let me know, otherwise this will be the basis for my server!

**Comment by Joulinar** [comment]
you can try using the following 2 lines to have it replaced in your `/etc/fstab` to get your 2 HDD mounted

```
UUID=F4A0F077A0F041A2 /mnt/volume/1TB ntfs-3 noatime,lazytime,rw,nofail,noauto,x-systemd.automount
UUID=41428830-31b0-44af-8a2d-ca7bacca9992 /mnt/volume/4TB ext4 noatime,lazytime,rw,nofail,noauto,x-systemd.automount
```

Once Drive Manager has been fixed, it will be much easier for you to have your HDD managed. But there we would need to wait on next bug fix release v6.30

**Comment by WolfShadowheart** [comment]
> 
> 
> you can try using the following 2 lines to have it replaced in your `/etc/fstab` to get your 2 HDD mounted
> 
> ```
> UUID=F4A0F077A0F041A2 /mnt/volume/1TB ntfs-3 noatime,lazytime,rw,nofail,noauto,x-systemd.automount
> UUID=41428830-31b0-44af-8a2d-ca7bacca9992 /mnt/volume/4TB ext4 noatime,lazytime,rw,nofail,noauto,x-systemd.automount
> ```
> 
> Once Drive Manager has been fixed, it will be much easier for you to have your HDD managed. But there we would need to wait on next bug fix release v6.30

Hey, thanks so much! Because the path for /mnt/volume/1TB was functional, I left that line in fstab, however I did edit /mnt/volume/4TB since it was not properly working. 

After inputting this and rebooting, the Plex Media Server does indeed let me select the mounted locations - however I am also met with this error.

`[853.587859] EXT4-fs error (device sdb1): ext4_find_entry:1455: inode  #28180499: comm Plex Media Serv: reading directory lblock 0`

Thanks so much for your help!

**Comment by Joulinar** [comment]
Hard to say what can cause this error message, could be something on the drive. Did you use an external power supply? Maybe you can backup all the data on it and try to reformat the entire drive.

**Comment by WolfShadowheart** [comment]
> 
> 
> Hard to say what can cause this error message, could be something on the drive. Did you use an external power supply? Maybe you can backup all the data on it and try to reformat the entire drive.

I have managed to resolve the issue thankfully, I don't exactly know what solved it however the drive will now be able to read files after being idle. This is on the same install from yesterday, I have still not used drive-manager. there is a line in TMPFS

`PARTUUID=6c586e13-02 / ext4 noatime,lazytime,rw 0 1`
`PARTUUID=6c586e13-01 /boot vfat noatime,lazytime,rw 0 2`
`/var/swap none swap sw 0 0`
`/var/log tmpfs size=50M,noatime,lazytime,nodev,nosuid,mode=1777`

That should be in line with what your solution is for the bug found as #3511, and then for my solution and anyone else who is looking to permanently mount external hard drives *once they have been mounted to the location manually! NOT using drive manager!*

`UUID=F4A0F077A0F041A2 /mnt/volume/1TB ntfs noatime,lazytime,rw,nofail,noauto,x-systemd.automount`
`UUID=41428830-31b0-44af-8a2d-ca7bacca9992 /mnt/volume/4TB ext4 noatime,lazytime,rw,nofail,noauto,x-systemd.automount` 

The issue was not enough sleep and a typo in my . Luckily, the above code should be able to be used, replacing the UUID with their own and following it, where it is to be mounted. What I did was created a folder inside /mnt called volume, and inside volume I created the folder 1TB and 4TB. 

`/mnt/volume/1TB` and `/mnt/volume/4TB`

I then went inside 1TB and mounted the 1TB hard drive, and did the same for the 4TB hard drive. With those mounted, `sudo df -h` shows that they exist with my test media. `sudo reboot` and repeating the command showed that they were mounted upon reboot. 

I very much appreciate your help, I'm very sorry for being so verbose in my issues (and being fairly off-topic...) but it really has helped me quite a bit. 

I have been able to repeat this successful bug work around and mounting solution on a fresh install of DietPi, by the way, so no need to use driver manager for now. I will update is errors occur after more than one day. 

Edit: As I was typing that I was going through another fresh install just to practice from flash to mount. It went fine, the only thing is that when mounting the hard drives, keeping the options in fstab didn't allow for them to mount automatically. However, changing the options to defaults 0 0 did. 

At this point I'm feeling more confident in getting USB's permanently mounted with fstab. Definitely committed a lot to memory through this!

**Comment by ghost** [comment]
Hi all,
Not sure if I have the same issue. I'm on an Odroid HC1, not an R-Pi.
As I said on the DietPi forum thread I was following [this youtube tutorial](https://www.youtube.com/watch?v=0QdBFpEzwM0&) on moving DietPi to the onboard sata drive on an Odroid HC1.

It was written for the older Stretch image and I am now trying it with the newer DietPi_OdroidN1-ARMv8-Buster image and have run into a problem. There were a few small differences, notably in the fstab file (PARTUUID in place of UUID for the entries of the partitions). 
But my problem was at the 29minute mark of the clip when I attempted to
`mkdir /mnt/data`
and received an error
`mkdir: cannot create directory ‘/mnt/data’: Read-only file system`
I just tried the workaround above that seems to be working for most people but received an error after the first command.
`
root@DietPi:~# mount -o remount,rw /
mount: /: can't find PARTUUID=1d034067-8e42-4797-b074-20dc677cbec8.`
see also;
```
root@DietPi:~# df -h
Filesystem      Size  Used Avail Use% Mounted on
udev            930M     0  930M   0% /dev
tmpfs           200M  6.4M  194M   4% /run
/dev/sda1       9.8G  575M  8.8G   7% /
tmpfs           998M     0  998M   0% /dev/shm
tmpfs           5.0M     0  5.0M   0% /run/lock
tmpfs           998M     0  998M   0% /sys/fs/cgroup
tmpfs           1.0G  8.0K  1.0G   1% /tmp
tmpfs            50M  8.0K   50M   1% /var/log
/dev/mmcblk1p1  250M   60M  191M  24% /boot
```
and
```
root@DietPi:~# blkid
/dev/mmcblk1p1: SEC_TYPE="msdos" LABEL_FATBOOT="boot" LABEL="boot" UUID="96C3-9298" TYPE="vfat" PARTUUID="000f1766-01"
/dev/mmcblk1p2: LABEL="rootfs" UUID="e139ce78-9841-40fe-8823-96a304a09859" TYPE="ext4" PARTUUID="000f1766-02"
/dev/sda1: LABEL="rootfs" UUID="1d034067-8e42-4797-b074-20dc677cbec8" TYPE="ext4" PARTLABEL="Linux filesystem" PARTUUID="dfe53edb-9859-4b7e-bca5-80303267a071"
/dev/sda2: UUID="F324-1EF1" TYPE="vfat" PARTLABEL="Linux filesystem" PARTUUID="e28dbe67-906f-4780-aaa1-1bbeb951af0c"
/dev/mmcblk1: PTUUID="000f1766" PTTYPE="dos"
```


```
root@DietPi:~# sudo nano /etc/fstab
Unable to create directory /root/.local/share/nano/: No such file or directory
It is required for saving/loading search history or cursor positions.

Press Enter to continue

```
My fstab looks like this (but I can't edit it);
```
# Please use "dietpi-drive_manager" to setup mounts
#----------------------------------------------------------------
# NETWORK
#----------------------------------------------------------------


#----------------------------------------------------------------
# TMPFS
#----------------------------------------------------------------
tmpfs /tmp tmpfs size=1024M,noatime,lazytime,nodev,nosuid,mode=1777

#----------------------------------------------------------------
# MISC: ecryptfs, vboxsf (VirtualBox shared folder), bind mounts
#----------------------------------------------------------------


#----------------------------------------------------------------
# SWAPFILE
#----------------------------------------------------------------


#----------------------------------------------------------------
# PHYSICAL DRIVES
#----------------------------------------------------------------
PARTUUID=1d034067-8e42-4797-b074-20dc677cbec8 / ext4 noatime,lazytime,rw 0 1
PARTUUID=000f1766-01 /boot vfat noatime,lazytime,rw 0 2
/var/swap none swap sw 0 0
tmpfs /var/log tmpfs size=50M,noatime,lazytime,nodev,nosuid,mode=1777
```

Thanks

**Comment by Joulinar** [comment]
@HC1-Dude 
I don't think this is related as yours might be self made due to the YouTube tutorial you followed. Basically as you are on nonRPi, you could have used Drive Manager to move rootFS to an external HDD. Let's have it fixed. Pls can you post output of

```
lsblk -Sfo +size
lsblk 
blkid
```

**Comment by ghost** [comment]
Thanks, I'll start over if need be but in the meantime;

```
root@DietPi:~# lsblk -Sfo +size
NAME HCTL       TYPE VENDOR   MODEL               REV TRAN FSTYPE LABEL UUID FSAVAIL FSUSE% MOUNTPOINT  SIZE
sda  0:0:0:0    disk ST2000LM ST2000LM015-2E8174 3101 usb                                               1.8T
```


```
root@DietPi:~# lsblk
NAME        MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
sda           8:0    0  1.8T  0 disk 
├─sda1        8:1    0   10G  0 part /
└─sda2        8:2    0  1.8T  0 part 
mmcblk1     179:0    0 14.9G  0 disk 
├─mmcblk1p1 179:1    0  257M  0 part /boot
└─mmcblk1p2 179:2    0 14.6G  0 part 
```


```
root@DietPi:~# blkid
/dev/mmcblk1p1: SEC_TYPE="msdos" LABEL_FATBOOT="boot" LABEL="boot" UUID="96C3-9298" TYPE="vfat" PARTUUID="000f1766-01"
/dev/mmcblk1p2: LABEL="rootfs" UUID="e139ce78-9841-40fe-8823-96a304a09859" TYPE="ext4" PARTUUID="000f1766-02"
/dev/sda1: LABEL="rootfs" UUID="1d034067-8e42-4797-b074-20dc677cbec8" TYPE="ext4" PARTLABEL="Linux filesystem" PARTUUID="dfe53edb-9859-4b7e-bca5-80303267a071"
/dev/sda2: UUID="F324-1EF1" TYPE="vfat" PARTLABEL="Linux filesystem" PARTUUID="e28dbe67-906f-4780-aaa1-1bbeb951af0c"
/dev/mmcblk1: PTUUID="000f1766" PTTYPE="dos"
```

**Comment by MichaIng** [comment]
@HC1-Dude 
I think you have applied the wrong UUID/PARTUUID to /etc/fstab. The error was RPi-only and so was the fix.

Please to the following:
- Edit `/etc/fstab`
- Replace
```
PARTUUID=1d034067-8e42-4797-b074-20dc677cbec8 / ext4 noatime,lazytime,rw 0 1
```
with
```
UUID=e139ce78-9841-40fe-8823-96a304a09859 / ext4 noatime,lazytime,rw 0 1
```
- And replace
```
PARTUUID=000f1766-01 /boot vfat noatime,lazytime,rw 0 2
```
with
```
UUID=96C3-9298 /boot vfat noatime,lazytime,rw 0 2
```

Then do:
```
umount /boot
mount -o remount,rw /
mount /boot
```

**Comment by ghost** [comment]
@MichaIng - thanks for the response but I can't edit /etc/fstab as its residing on a read only filesystem. I think I'll have to just start over. Thanks for your response though.

**Comment by MichaIng** [comment]
@HC1-Dude 
Ah sorry run the following first:
```
mount -o remount,rw /
```

**Comment by Hukuma1** [comment]
Rebooted after installing a router and got hit with this. Luckily found this via Google. Thank you! Couldn't update to v6.30 (or much anything else) until doing the manual fix.

**Comment by mustafababil** [comment]
I had this issue but since I just met with DietPi in version v6.29.2, I thought pure reason was my memory card or RPi 2B, and I was applying workaround solution already.

Today I became aware of this bug while reading the changelock. Updated immediately and it solved the problem. Thanks for the support.

**Comment by MichaIng** [comment]
Good to hear that the patch to solve it during update works fine 🙂.
